lib/i18n-js/listen.rb

```diff
@@ -7,7 +7,8 @@ class << self
 
   def self.listen(
     config_file: Rails.root.join("config/i18n.yml"),
-    locales_dir: Rails.root.join("config/locales")
+    locales_dir: Rails.root.join("config/locales"),
```

Comment:
I guess this option could be now deprecated in favor of `locales_dirs` but it can be done in separate PR